bin/templates/scripts/cordova/lib/check_reqs.js

```diff
@@ -121,6 +121,20 @@ module.exports.check_cocoapods = function (toolChecker) {
         // check whether the cocoapods repo has been synced through `pod repo` command
         // a value of '0 repos' means it hasn't been synced
         .then(function (toolOptions) {
+            let podVersion = shell.exec('pod --version', { silent: true });
+            if (podVersion) {
+                const semver = podVersion.toString().split('.');
+                if (semver.length >= 3) {
+                    const major = parseInt(semver[0]);
+                    const minor = parseInt(semver[1]);
+                    const patch = parseInt(semver[2]);
+
+                    // starting with 1.8.0 cocoapods now use cdn and we dont need to sync first
+                    if ((major >= 1 && minor >= 8 && patch >= 0) || toolOptions.ignore) {
```

Comment:
Hi,

maybe it's better this way:

`if (((major > 1) || (major == 1 && minor >= 8 && patch >= 0)) || toolOptions.ignore) {`

do not you think?

Reply:
It's maybe also sufficient to only check for major and minor version.